_posts/2022-11-17-what-makes-a-python-package-healthy.md

```diff
@@ -0,0 +1,536 @@
+
+<i class="fas fa-info-circle"></i> The code for many open source software packages can be found on [GitHub](https://www.github.com). GitHub is a free-to-use website that runs `git` which is a version control system. Version control allows developers to track historical changes to code and files. GitHub which is a website allows developers to communicate openly, review new code changes and update content in a structured way. 
+{: .notice }
+
+### What does GitHub (and Ivan) think about health checks for Python open source software?
+
+Ivan, is a long time advisor, editor and member of the pyOpenSci community. He's also a generally a great human being who is growing open science efforts such as 
+[Open Science Labs](https://opensciencelabs.org/); which is a gloabl community devoted to education efforts and tools that support open science.  
+
+Ivan was quick to point out some basic metrics offered by GitHub which follow their 
+[community standards online guidebook here](https://opensource.guide/).
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Maybe not totally related, but github has a section called community standards that mabe could be used as reference, for example: <a href="https://t.co/wmu1bDdcQR">https://t.co/wmu1bDdcQR</a></p>&mdash; XMN (@xmnlab) <a href="https://twitter.com/xmnlab/status/1578745372880429057?ref_src=twsrc%5Etfw">October 8, 2022</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script> 
+
+Actually it's totally related, Ivan! Let's have a look look at the [pyOpenSci contributing-guide GitHub repository](https://www.github.com/pyopensci/contributing-guide/community) to see how we are 
+doing as an organization.
+
+Note that we are missing some important components:
+
+* A code of conduct
+* A contributing file that helps people understand how to contribute 
+* Issue template for people opening issues 
+* Pull request templates to guide people through opening pull requests 
+* Repository admins accepting content reporting 
+
+{% include figure image_path="/images/python-software-health/healthy-software-repo-file-checks-github.png" alt="Image showing the community standards page in GitHub. You can see in the image we are missing several critical files including  a code of conduct file, a contributing file that helps people understand how to contribute to the guide and issue and pull request templates." caption="Here you can see the community page in GitHub for our contributing-guide repository. Note that we are missing several important items in the repo including a code of conduct file, a contributing file that helps people understand how to contribute to the guide and issue and pull request templates. HELP!" %}
+
+> Um.... we've got some real work to do, y'all on our guides and repos. We need to set a better example here don't we? We welcome help welcome if you are reading this and wanna contribute. Just sayin... 
```

Comment:
Should we raise issues, "Add CoC", "Add contributing.md", "Add pull request templates"?  
I would just raise them but I'm not quite sure where you want them.  
Do we want a separate governance repo with the CoC and contributing that we can link to from every other repo?

Reply:
i'm not sure how this relates to this blog post david. it seems like a comment that belongs elsewhere. i'm just making the point that our repos don't have the files if you look at that link we are missing a bunch of green checks.